addons/website_sale/tests/test_customize.py

```diff
@@ -2,14 +2,15 @@
 
 import base64
 
-from odoo.addons.base.tests.common import HttpCaseWithUserDemo, HttpCaseWithUserPortal
 from odoo.fields import Command
 from odoo.tests import tagged
 from odoo.tools.misc import file_open
 
+from odoo.addons.base.tests.common import HttpCaseWithUserDemo, HttpCaseWithUserPortal
+
 
 @tagged('post_install', '-at_install')
-class TestUi(HttpCaseWithUserDemo, HttpCaseWithUserPortal):
+class TestCustomize(HttpCaseWithUserDemo, HttpCaseWithUserPortal):
 
     def setUp(self):
```

Comment:
setUpClass? Didn't check all the files, but I think it could be changed in more than just this one